src/sidebar/index.js

```diff
@@ -140,10 +141,10 @@ import { Injector } from '../shared/injector';
 /**
  * Launch the client application corresponding to the current URL.
  *
- * @param {object} config
+ * @param {import('../types/config').SidebarSettings} sidebarSettings
```

Comment:
`sidebarSettings` is perhaps a bit verbose. `settings` would be fine here I think, since that is what the object is called that is exposed to most other code in the app.

Reply:
That seems fine; I'll shorten.